### PR TITLE
chore: add nil checks and tweaks to gRPC handlers

### PR DESCRIPTION
**Pull request by alexanderbez**
Closes: #1224

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: x/pool-incentives/keeper/grpc_query.go [hunk 1]
```diff
@@ -92,9 +93,13 @@ func (q Querier) IncentivizedPools(ctx context.Context, _ *types.QueryIncentiviz
 	}, nil
 }
 
-// ExternalIncentiveGauges iterates over all gauges, returns gauges externally incentivized,
-// excluding default gagues created with pool.
+// ExternalIncentiveGauges iterates over all gauges, returns gauges externally
+// incentivized, excluding default gauges created with pool.
 func (q Querier) ExternalIncentiveGauges(ctx context.Context, req *types.QueryExternalIncentiveGaugesRequest) (*types.QueryExternalIncentiveGaugesResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
```
Comment (p0mvn) [inline x/pool-incentives/keeper/grpc_query.go:100]: nit: I see this error message and a few others being reused several times, consider extracting constants

**Review by p0mvn — APPROVED** [review]
LGTM

**Comment by ValarDragon** [comment]
LGTM!

**Comment by alexanderbez** [comment]
@ValarDragon I'm not 100% sure. These methods come from the gRPC service generator. But something tells me the answer is no lol.

**Pull request merged**